crypto/bn/bn_exp.c

```diff
@@ -240,7 +240,6 @@ int BN_mod_exp_recp(BIGNUM *r, const BIGNUM *a, const BIGNUM *p,
                                  * when there is only the value '1' in the
                                  * buffer. */
     wstart = bits - 1;          /* The top bit of the window */
-    wend = 0;                   /* The bottom bit of the window */
```

Comment:
I'd rather we didn't lose the comment explaining what `wend` is

Reply:
Moved the comment to the next usage of `wend`